package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Kinvey, Inc.",
   "scripts": {
     "build": "lerna run build --parallel",
-    "test": "mocha --require babel-polyfill --require babel-register 'src/**/*.spec.js'"
+    "test": "mocha --require babel-polyfill --require babel-register src/**/*.spec.js"
```

Comment:
This only runs about half the unit tests on MacOS. The solution is use `\"src/**/*.spec.js\"` instead.

Reply:
I'm still not comfortable deciding on this, since neither of us knows what is happening :D

Reply:
Can we make the change so unit tests continue to work on Windows and MacOS environments? Then we can spend time learning the why.

Reply:
Sure, but this isn't the place for it. This stuff isn't going to get merged into master for months. I investigated it a bit. I think it comes down to the library mocha uses to expand glob patterns. In the mocha docs it's with double quotes, so I think we're good on that front. We should make the change in the important branches - master, I guess.

Reply:
But if I or someone else was to checkout this branch, work on the code, add unit tests, etc. and then try to run unit tests it won't work. That is why we need to change it here.

Reply:
Yes, we need to do it in both places, because each is broken for one of us :D